### PR TITLE
Add black and isort

### PR DESCRIPTION
**Pull request by kasium**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,75 +1,45 @@
...
+            "Programming Language :: Python :: 3.10",
+            "Topic :: Scientific/Engineering :: Visualization",
+            "Topic :: Software Development :: Libraries :: Python Modules",
+            "Topic :: Database",
         ],
-    },
-)
+        keywords="sql relational databases ER diagram render",
+        packages=["eralchemy2"],
+        extras_require={"dev": ["black==22.8.0", "isort==5.10.1"]},
```
Comment (maurerle) [inline setup.py:40]: is there a specific reason to have this version fixed?
If people want to use other packages which rely on a different version of black or isort next to it they will have a hard time figuring out the dependencies

Comment (kasium) [reply setup.py:40]: Do you expect that people install this package with dev dependencies?

IMHO, this normally only happens if you work in the project and the you are in a separated environment.

Also, if you add a CI at one point and black/isort updates, then the CI might fail out of the sudden

Comment (maurerle) [reply setup.py:40]: Having a CI fail on a future version is an indicator that some feature we are using will be deprecated in the future and we need a change in the future anyway. So this is more like a feature than a big in my point of view.
Having tests run in CI with the latest version is a feature to see if it still works as I don't update my environment everytime.

In my opinion it's better to only have fixed versions if this is required due to incompatibilities, and I don't want to change the extras_require every time just to update the version.

Those discussions are getting quite philosophically fast, and I said that especially for dev-reqs I don't think that they have to be fixed, while you are saying that especially for dev-reqs everyone would have a separate env anyway.

I don't know if there is a pythonic consensus on how to handle this, but I know that spyder and flask do it that way and did not find a major project having fixed version dependencies.

Comment (kasium) [reply setup.py]: I removed the pins

**Comment by kasium** [comment]
@maurerle since I stumbled across that: there is now requirements.txt, test_requirements.txt, setup.py - install_requires and setup.py - extras_require. It's maybe a good think to either use setup.py or *.txt files (just an idea). Also a CI would be a great thing!

**Comment by maurerle** [comment]
Can you run black again on current master to resolve the conflicts?

**Comment by maurerle** [comment]
Thanks a lot for your contributions! :)
And yes, I will remove the requirements.txt then after merging this.

I will also rename the main branch and then add a CI for the tests. But I am also open for contributions regarding github workflows.

**Pull request merged**